test/unit/models/flag.test.js

```diff
@@ -0,0 +1,46 @@
+const chai = require("chai");
+const { expect } = chai;
+const chaiHttp = require("chai-http");
+
+const addUser = require("../../utils/addUser");
+const cleanDb = require("../../utils/cleanDb");
+
+const flagModel = require("../../../models/flag");
+const firestore = require("../../../utils/firestore");
+const flagFirestore = firestore.collection("featureFlags");
+
+const flagData = {
+  title: "dark - mode",
+  enabled: true,
+  roles: ["members", "no_role", "super_user"],
+  users: {
+    "pallab id": true,
+    "rohit id": false,
+  },
+};
+
+chai.use(chaiHttp);
```

Comment:
We are not using `chaiHttp` in unit tests. Can we remove this and the import header?